examples/seismic/tutorials/16_ader_fd.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "31226b14-e907-4503-9918-a09163363217",
```

Comment:
<p>&gt;This process is iterated to obtain the required temporal derivatives.</p><p> </p><p>You mean "this process can be iterated for higher order equations?" maybe?</p><p> </p><p>So this is indeed how you usually derive 4th order equations for 2nd order scalar (tti)/acoustic</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2450/discussion/#comment-1758172527'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>